### PR TITLE
Separate user and admin

### PR DESCRIPTION
**Pull request by andrewseamanco**
Create new admin filter and folder to separate user and admin flows

### CODE REVIEW

File: bait-files/src/main/java/com/google/AdminFilter.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  @Override
+  public void init(FilterConfig config) throws ServletException {}
+
+  @Override
+  public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
+      throws IOException, ServletException {
+
+        HttpServletRequest request = (HttpServletRequest) req;
+        HttpServletResponse response = (HttpServletResponse) res;
+        HttpSession session = request.getSession(false);
```
Comment (tambchop) [inline bait-files/src/main/java/com/google/AdminFilter.java:37]: I'd include an inline comment saying what the name of the parameter is that "false" is representing.

So like:
request.getSession(\* whateverThisParameterIsCalled = */ false);

File: bait-files/src/main/java/com/google/AdminFilter.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            return;
+        }
+    }
+
+    public Permission getCurrentUserPermission() {
+    List<User> allUsers = ObjectifyService.ofy().load().type(User.class).list();
+    return allUsers.stream().filter(user -> 
+      user.getUserId().equals(UserServiceFactory.getUserService().getCurrentUser().getUserId()))
+    .collect(onlyElement())
+    .getPermission();
```
Comment (tambchop) [inline bait-files/src/main/java/com/google/AdminFilter.java:54]: Formatting: fix the indentation for this.

File: bait-files/src/main/java/com/google/LoginFilter.java [hunk 3]
```diff
@@ -70,17 +70,15 @@ else if (!userIsRegistered(userService.getCurrentUser().getUserId())
...
-        return;
-        //Case: User is trying to access ADMIN features of the application
-      } else if (request.getRequestURI().endsWith("requests.html")) {
-          if (getCurrentUserPermission() == Permission.ADMIN) {
-              chain.doFilter(req, res);
-          } else {
-            RequestDispatcher requestDispatcher = request.getRequestDispatcher("WEB-INF/profile.jsp");
-            requestDispatcher.forward(request, response);  
-          }
+              if (getCurrentUserPermission()==Permission.USER) {
```
Comment (tambchop) [inline bait-files/src/main/java/com/google/LoginFilter.java:73]: Formatting: add spacing around the double equals.

File: bait-files/src/main/java/com/google/AdminFilter.java (hunk 1 above)
Comment (andrewseamanco) [reply bait-files/src/main/java/com/google/AdminFilter.java:37]: Added inline comment

File: bait-files/src/main/java/com/google/AdminFilter.java (hunk 2 above)
Comment (andrewseamanco) [reply bait-files/src/main/java/com/google/AdminFilter.java]: Fixed

File: bait-files/src/main/java/com/google/LoginFilter.java (hunk 3 above)
Comment (andrewseamanco) [reply bait-files/src/main/java/com/google/LoginFilter.java]: Added spacing

**Pull request merged**